messages_test.go

```diff
@@ -8,20 +8,17 @@ import (
 	"testing"
 
 	"github.com/sashabaranov/go-openai"
+	"github.com/sashabaranov/go-openai/internal/test"
 	"github.com/sashabaranov/go-openai/internal/test/checks"
 )
 
 var emptyStr = ""
 
-// TestMessages Tests the messages endpoint of the API using the mocked server.
-func TestMessages(t *testing.T) {
+func registerServer(t *testing.T, server *test.ServerTest) {
```

Comment:
split func for linter

Reply:
The `registerServer` name is very generic, let's name it with something more concrete for this test

Reply:
rename to `setupServerForTestMessage` 
[8facd6e](https://github.com/sashabaranov/go-openai/pull/799/commits/8facd6e65b6b08056aa32b20db0b5b7e38fbd932)